packages/workbench/src/integration.css

```
@@ -0,0 +1,15 @@
+/* Integration version - no Tailwind import, processed by app's Tailwind */
+/* This file should be imported in the app's CSS file (e.g., globals.css) */
+/* The app must have Tailwind imported before importing this file */
+
+/* Tell Tailwind to scan workbench components for class names */
+/* Note: The path below is relative to this CSS file's location in the package */
+@source "./**/*.{ts,tsx}";
+
+@import './base.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)